index.html

```diff
@@ -20,160 +20,165 @@
   </head>
 
   <body>
-    <div class="header">
+    <section class="header">
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <section>
```

Comment:
Like the `<header>`, we also have a dedicated element for navigation called [`<nav>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)